packages/framework/src/Migrations/Version20200219145345.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace App\Migrations;
+namespace Shopsys\FrameworkBundle\Migrations;
 
 use Doctrine\DBAL\Schema\Schema;
 use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
```

Comment:
I suggest wrapping the migration in `up` method with condition
```
if ($this->isAppMigrationNotInstalledRemoveIfExists('Version20200219145345')) {
    $this->sql('ALTER TABLE adverts ADD datetime_visible_from TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL');
    $this->sql('ALTER TABLE adverts ADD datetime_visible_to TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL');
}
```

that way the migration file may be safely deleted during upgrade and it's not necessary to check whether it's installed or not. 
So then the second bullet in the upgrade notes may be deleted

Reply:
hmm 🤔 that did not cross my mind. [Migration changed](https://github.com/shopsys/shopsys/pull/2975/commits/204d2ba7381481afc8093ec290a29a1089c3ab5a) and [upgrade log](https://github.com/shopsys/shopsys/pull/2975/commits/a5b6d30d9791638b02cbaac8dadd721ffc93d463) too